ephem/doc/quick.rst

```diff
@@ -840,6 +840,20 @@ local time
  * The ``localtime()`` function converts a PyEphem date
    into a Python ``datetime`` object expressed in your local time zone.
 
+to timezone
+-----------
+
+ >>> d = ephem.Date('1997/3/9 5:13')
+ >>> ephem.to_timezone(d, ephem.UTC())
```

Comment:
Could we switch `ephem.UTC` to being an instance, so that users don't have to incur the expense of instantiating it for use and could drop the `()`?

Reply:
sure

Reply:
Excuse me for making this a teachable moment (with you as the teacher), but does that mean, instead of

    class UTC(_tzinfo):
        ...

you want to see 

    UTC = _tzinfo(_timedelta())

or

    UTC = _tzinfo(_timedelta(0))

?